feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.html

```diff
@@ -1,3 +1,7 @@
+<div *ngIf="!attribute.visible" class="cx-hidden-msg">
+  <cx-icon [type]="iconTypes.WARNING" aria-hidden="true"></cx-icon>
+  {{ 'configurator.attribute.notVisibleAttributeMsg' | cxTranslate }}
+</div>
```

Comment:
shouldn't this only be displayed in case expMode=true?

Reply:
Normally only visible attributes come, thus `This attribute is not visible to the user` message will not be displayed.
There is no need to check additionally whether the expert mode is set